script.js

```diff
@@ -1,11 +1,23 @@
 // Create an array called `colors` that contains three colors.
+var colors = ["red", "yellow", "green"];
 
 // Log the last item in the array to the console.
+console.log(colors[2]);
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for (var i = 0; i < colors.length; i++) {
+	console.log("My favorite color is " + colors[i]);
+}
 
 // Create an `instructor` object that contains three key-value pairs.
+var instructor = {
+	keyOne: "valueOne",
+	keyTwo: "valueTwo",
+	keyThree: "valueThree",
+}
 
```

Comment:
👍 technically correct, but in general would like to see more semantic key-value pairs